icicle/curves/bls12_381/params.cuh

```diff
@@ -190,4 +190,11 @@ namespace PARAMS_BLS12_381{
                                                                           0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000};
   static constexpr storage<fq_config::limbs_count> weierstrass_b_g2_im = {0x00000004, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000,
                                                                           0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000, 0x00000000};
-}
\ No newline at end of file
+
+  constexpr storage<fq_config::limbs_count> fq_config::m;
+  constexpr storage<fq_config::limbs_count> fq_config::modulus;
+  constexpr storage<2*fq_config::limbs_count> fq_config::modulus_wide;
+  constexpr storage<fp_config::limbs_count> fp_config::m;
+  constexpr storage<fp_config::limbs_count> fp_config::modulus;
+  constexpr storage<2*fp_config::limbs_count> fp_config::modulus_wide;
```

Comment:
@gkigiermo I think we can remove these if we require c++17 standard in the CMakeLists for both [CXX and CUDA](https://github.com/ingonyama-zk/icicle/blob/main/icicle/CMakeLists.txt#L4-L5) which would require changing the minimum cmake version to 3.18.

Reply:
You are right; I just did it.